x11-wm/pekwm/pekwm-0.2.1.ebuild

```
@@ -0,0 +1,86 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake-multilib desktop optfeature
+
+DESCRIPTION="A lightweight window manager initially based on aewm++"
+HOMEPAGE="
+	https://www.pekwm.se/
+	https://github.com/pekdon/pekwm
+"
+SRC_URI="
+	https://github.com/pekdon/${PN}/releases/download/release-${PV}/pekwm-${PV}.tar.gz
+"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~mips ~ppc ~x86"
+IUSE="debug +jpeg +png truetype +xft xinerama +xpm"
+
+RDEPEND="
+	png? ( media-libs/libpng:0 )
+	jpeg? ( virtual/jpeg:0 )
+	xpm? ( x11-libs/libXpm )
+	x11-libs/libX11
+	x11-libs/libXext
+	truetype? ( x11-libs/libXft )
+	xinerama? ( x11-libs/libXinerama )"
```

```suggestion
	x11-libs/libX11
	x11-libs/libXext
	jpeg? ( virtual/jpeg:0 )
	png? ( media-libs/libpng:0 )
	truetype? ( x11-libs/libXft )
	xinerama? ( x11-libs/libXinerama )
	xpm? ( x11-libs/libXpm )"
```

We like to sort it this way.
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Sorting_dependencies